component/src/main/resources/installation-scripts/install.sh

```diff
@@ -31,28 +31,38 @@ ballerina_lib_location=$ballerina_home/bre/lib
 ballerina_balo_location=$ballerina_home/lib/repo
 version=${project.version}
 module_name=kafka
+fileNamePattern="wso2-kafka-module-*.*.*.jar"
 
-fileNamePattern="wso2-kafka*.jar"
 for filename in $ballerina_home/bre/lib/*; do
     existingFile=${filename##*/}
     [[ $existingFile == $fileNamePattern ]] && file=$existingFile || file=""
     if [ "$file" != "" ]; then
-        rm $ballerina_lib_location/$existingFile
+        echo "[WARNING] Another version of Kafka module is already installed."
+        read -r -p "Do you want to uninstall it? (Y/N): " response
+        if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]; then
 
-        if [ -e "$ballerina_lib_location/$existingFile" ]; then
-            echo "Error occurred while deleting dependencies from $ballerina_lib_location"
-            echo "Please manually delete $ballerina_lib_location/$existingFile and $ballerina_balo_location/wso2/$module_name/0.0.0/$module_name.zip"
-            exit 1
-        fi
+            rm $ballerina_lib_location/$existingFile
+
+            if [ -e "$ballerina_lib_location/$existingFile" ]; then
+                echo "[ERROR] Error occurred while deleting dependencies from $ballerina_lib_location"
+                echo "Please manually delete $ballerina_lib_location/$existingFile and $ballerina_balo_location/wso2/$module_name/0.0.0/$module_name.zip"
+                exit 1
+            fi
 
-        rm -r $ballerina_balo_location/wso2/$module_name/0.0.0
+            rm -r $ballerina_balo_location/wso2/$module_name/0.0.0
 
-        if [ -e "$ballerina_balo_location/wso2/$module_name/0.0.0/$module_name.zip" ]; then
-            echo "Error occurred while deleting $module_name balo from $ballerina_balo_location"
-            echo "Please manually delete $ballerina_balo_location/wso2/$module_name/0.0.0 directory"
-            exit 2
+            if [ -e "$ballerina_balo_location/wso2/$module_name/0.0.0/$module_name.zip" ]; then
+                echo "[ERROR] Error occurred while deleting $module_name balo from $ballerina_balo_location"
+                echo "Please manually delete $ballerina_balo_location/wso2/$module_name/0.0.0 directory"
+                exit 2
+            else
+                echo "[INFO] Successfully uninstalled existing Kafka module: $existingFile"
+            fi
+        elif [[ $response =~ ^([nN][oO]|[nN])$ ]]; then
+            echo "[INFO] Installing module without uninstall the existing version of Kafka module: [$existingFile]."
```

Comment:
We should exit the installation here, because installing the connector while there is an existing version is not supported.